docs/nvidia-guide.md

```diff
@@ -0,0 +1,78 @@
+# NVIDIA graphics cards
+_Driver API and source code available in [`liquidctl.driver.nvidia`](../liquidctl/driver/nvidia.py)._
+
+Support for these cards in only available on Linux.
+
+Additional requirements must also be met:
+
+- optional Python dependency `smbus` is available
+- `i2c-dev` kernel module has been loaded
+- specific unsafe features have been opted in
+- r/w permissions to card-specific `/dev/i2c-*` devices
+
+---
+
+Jump to a specific card:
+
+* _Series 10/Pascal:_
+    - [EVGA GTX 1080 FTW](#evga-gtx-1080-ftw)
+<!-- - [EVGA GTX 1070 FTW](#evga-gtx-1070-1080-ftw) -->
+<!-- - [EVGA GTX 1080 FTW](#evga-gtx-1070-1080-ftw) -->
```

Comment:
Is this line a copy paste error or are those commented out lines different?

Reply:
It's intentional.  In this example of adding support for the 1070 FTW,

```
- [EVGA GTX 1080 FTW](#evga-gtx-1080-ftw)
```

is replaced with

```
- [EVGA GTX 1070 FTW](#evga-gtx-1070-1080-ftw)
- [EVGA GTX 1080 FTW](#evga-gtx-1070-1080-ftw)
```

The idea is to have both cards of the same "family" (protocol/driver) linking to the same "EVGA GTX 1070/1080 subsection".